### ISSUE TITLE
How to pop PersistentTabBarView/BottomNavBar

### ISSUE THREAD

**Issue by aifrosals** [BilalShahid13/PersistentBottomNavBar#201]
How to pop the PersistentTabBarView page so that PersistentBottomNavBar disappears

**Comment by mahdi-ko** [comment]
I need to do this as well did @aifrosals did you figure it out?

**Comment by mahdi-ko** [comment]
This works for most scenarios, but not when you want to pop the nav bar  and display a new screen instead, so when back button is clicked it closes the app, I found a work around where I restart the whole app on logout, and I open the screen I wanted based on boolean value.

**Comment by tannatsri** [comment]
@mahdi-ko hey did you find any solutions for it?
I am also facing the same issue.

**Comment by mahdi-ko** [comment]
> @mahdi-ko hey did you find any solutions for it? I am also  facing the same issue.

yes I did, what you can do is to render the screen that contains the presistant bottom navigation bar based on boolean value, like this example:
`isAuth ? BottomNavigation() : GettingStarted();`
